### ISSUE TITLE
Support Mojave dark mode

### ISSUE THREAD

**Issue by sindresorhus** [Clipy/KeyHolder#19]
It currently doesn't adapt well to dark mode:

<img width="262" alt="screen shot 2018-07-20 at 12 51 40" src="https://user-images.githubusercontent.com/170270/42985554-247d79c0-8c1c-11e8-85f2-2ba80119eaa0.png">

This is how the system inputs look like in dark mode:

<img width="160" alt="screen shot 2018-07-20 at 12 56 38" src="https://user-images.githubusercontent.com/170270/42985612-68063dd0-8c1c-11e8-9c96-1a111b44e4d6.png">
<img width="171" alt="screen shot 2018-07-20 at 12 56 50" src="https://user-images.githubusercontent.com/170270/42985615-684d9f9a-8c1c-11e8-9c0b-167554b785b9.png">

**Comment by rkbhochalya** [comment]
Until [this](https://github.com/Clipy/KeyHolder/pull/26) pull request is merged (hopefully with some changes), one can override colors as following:
```swift
@IBOutlet weak var recordView: RecordView! {
    didSet {
        recordView.tintColor = .controlAccentColor
        recordView.backgroundColor = .controlBackgroundColor
        recordView.borderColor = .controlShadowColor
    }
}
```

This is how it will look like in light and dark mode after applying above colors:
![image](https://user-images.githubusercontent.com/12229032/56080150-f5f2c700-5e1a-11e9-85cc-3a0d9b2ba802.png)
![image](https://user-images.githubusercontent.com/12229032/56080145-e6737e00-5e1a-11e9-8ed7-191b5d03f3af.png)

**Comment by Econa77** [comment]
Fixes #26 👍
